README.md

```diff
@@ -26,38 +64,51 @@ packer {
 }
 ```
 
+#### Manual Installation
 
-#### Manual installation
-
-You can find pre-built binary releases of the plugin [here](https://github.com/hashicorp/packer-plugin-vmware/releases).
-Once you have downloaded the latest archive corresponding to your target OS,
-uncompress it to retrieve the plugin binary file corresponding to your platform.
-To install the plugin, please follow the Packer documentation on
-[installing a plugin](https://www.packer.io/docs/extending/plugins/#installing-plugins).
+You can download [pre-built binary releases][releases-vmware-plugin] of the
```

Comment:
We can actually replace this whole install process now. As of Packer 1.8.1 users can use `packer plugins install github.com/hashicorp/vsphere` to install the latest available version of this plugin. 

The command was released in Packer 1.8.0. But in 1.8.1 it was updated to work with JSON templates as well.

Reply:
Nice additional catch - will update accordingly. Once agreed we can update on the vSphere plugin README, too.

Reply:
Updated - ready for re-review.